library/ecp_curves.c

```diff
@@ -5477,6 +5477,204 @@ static int ecp_mod_p256k1(mbedtls_mpi *N)
 }
 #endif /* MBEDTLS_ECP_DP_SECP256K1_ENABLED */
 
-#endif /* !MBEDTLS_ECP_ALT */
+#if defined(MBEDTLS_TEST_HOOKS)
+
+/** Initialise a modulus with hard-coded const curve data.
```

Comment:
I was not certain weather the documentation is better fitted here or in `ecp_invasive.h`. I feel it would be more usefull at the source file, being a static function.

Reply:
It is static testable so the reader has a hint that it has a declaration in the header. I agree that it might be slightly inconvenient to have it there, but I think it is conceptually more sane.